docs/index.md

```diff
@@ -8,6 +6,7 @@ bugs at runtime.
 
 The analyzer works in Visual Studio 2015 Update 1 or later. Install one of these

```

Comment:
I think that's no longer true... Not sure if we should revert the changes or update the docs.
(I switched the analyzers to .NET Standard during my work to make the build pass on Linux. They were referencing an old version of Roslyn, which caused warnings in tests when targeting .NET Core)

Reply:
Let's merge anyway so that we can see how the docs look. We can fix this later.

Reply:
Yeah, I had some misgivings about that line, but came to the same conclusion you did.